src/main/frontend/components/property/config.cljs

```diff
@@ -503,13 +504,13 @@
    :block-below {:icon :layout-align-top :title "Below the block"}})
 
 (rum/defc ui-position-sub-pane
-  [property {:keys [id set-sub-open! _position]}]
+  [property {:keys [id set-sub-open! _ui-position]}]
   (let [handle-select! (fn [^js e]
                          (when-let [v (some-> (.-target e) (.-dataset) (.-value))]
-                           (db-property-handler/upsert-property!
-                            (:db/ident property)
-                            (assoc (:block/schema property) :position (keyword v))
-                            {:property-name (:block/title property)})
+                           (db-property-handler/set-block-property!
```

Comment:
One subtle change with using `set-block-property` is that the property entity's :block/updated-at is no longer updated. In this PR we've changed it so that only changing the name, description, property type or cardinality results in updating the property's updated-at. If we want to make it so that making any property config change updates the updated-at or something similar, I'm happy to make the change